app/src/Components/Event/EventSpeakers.tsx

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
This line should be removed (we shouldn't disable ES Lint checks unless there's a strong reason to)